app/models/tagging.rb

```diff
@@ -46,9 +46,11 @@ def update_taggings_count
   end
 
   def update_search
-    reindex_boundary = 100
-    if %w(Character Relationship Freeform).include?(tagger.type)
-      tagger.enqueue_to_index if tagger.taggings_count < reindex_boundary
-    end
+    return unless tagger && Tag::USER_DEFINED.include?(tagger.type)
+
+    # Reindex the tag for updated suggested tags, which give you an idea of
```

Comment:
Comment adapted from https://github.com/otwcode/otwarchive/pull/3395/files#r204898786.

Reply:
This comment (and the one it was adapted from) might be slightly misleading. From Tag Wrangling chairs:

> but sometimes when reviewing things that have a lot of use, people look over the suggested fandoms
> because they'll still appear in addition to the attached fandom for canonical/unfilterable tags whether a tag used in multiple fandoms needs to be disambiguated.

Reply:
How's this?
> Reindex the tag for updated suggested tags.
> Suggested tags help wranglers figure out new tag and disambiguate existing canonical/unfilterable tags in multiple fandoms.

Reply:
Hmm, that comment from the Tag Wrangling chairs makes it sound like the change to suggested fandoms is going to be a problem. (`suggested_parent_tags`, which is used to generate `@suggested_fandoms`, only looks at tags on the first 50 works in the tag. I think it picks them in increasing order by ID, so suggested fandoms wouldn't give any idea of what the recent usage looks like, and whether a pre-existing canonical character has the same name as a new character in another fandom.)

Reply:
I think that's a good comment, @redsummernight 

@tickinginstant, it sounds like the suggested fandom code will be okay: one of the chairs has now indicated wranglers are actually _not_ supposed to use suggested tags to check for ambiguity, but rather the works page -- most tags with more than one suggested fandom come from crossovers, not ambiguous usage.

Reply:
@redsummernight Oops, actually, I think a couple words went missing in your potential comment. Something like "figure out where to wrangle new tags and if it's necessary to disambiguate..."

Reply:
Previously:

    has_many :suggested_fandoms, -> { distinct }, through: :works, source: :fandoms

So it can be slow.

I'm wondering if we can use `WorkQuery` results/facets for the new `suggested_parent_tags` instead of that first-50 SQL query.

Reply:
Never mind, the list of suggested fandoms doesn't need to be exhaustive.

@sarken I edited the previous comment.

Reply:
Yeah, if the 50-work limit wasn't okay, I think the only reasonable approach would have been to go through Elasticsearch and aggregate the fandom tags. I'm relieved that that's unnecessary!